scripts/stack_tile.py

```diff
@@ -1,6 +1,3 @@
-import sys
-
-sys.path.append("./")
```

Comment:
Managed to run this script in the top-level directory (i.e. `model/`) using:

```
python scripts/stack_tile.py
```
which outputs:
```
Found 39 Sentinel-2 items
EPSG code based on Sentinel-2 item:  32643
Searching S1 in date range 2020-08-28/2020-09-03
Found 4 Sentinel-1 items
Most overlapped orbit:  descending
Found 4 DEM items
...
```
so probably don't need this sys.path.append line?